task_DEV-5/DEV-5/EntryPoint.cs

```diff
@@ -0,0 +1,19 @@
+﻿using System;
+using System.Collections.Generic;
+
+namespace DEV_5
+{
+    internal class EntryPoint
+    {
+        private static void Main(string[] args)
+        {
+            var list = new List<IFlyable> {new Bird(), new Plane(), new SpaceShip()};
+
+            foreach (var item in list)
+            {
+                item.FlyTo(new Point(100, 200, 800));
```

Comment:
please do not use magic numbers

Reply:
Sorry, what do words "magic numbers" mean?

Reply:
Numbers that are not assigned to variables and carry some meaning.
It is better to use  class fields for example minSpeed = 0 and maxSpeed = 20
xCoordinate, yCoordinate ...
startSpeed = 200 etc

Reply:
is it OK for you?